src/python/openvino_genai/__init__.py

```diff
@@ -28,8 +28,10 @@
 
 # LLM pipeline
 from .py_openvino_genai import (
-    LLMPipeline, 
-    draft_model
+    LLMPipeline,
+    draft_model,
+    prompt_lookup
```

Comment:
as we can just enable it via `openvino_genai.LLMPipeline(model, device, prompt_lookup=True)` where `prompt_lookup` is kwarg, we don't see explicitly property in Python (as we don't have it for simple properties of generation config)